index.html

```diff
@@ -50,39 +49,64 @@ <h3>Furahia Insurance <br><span class="logo-writing2">Since 1965</span></h3>
+  <main>
+    <article>
+      <h1 id="intro2">PROMOTING A LIFE <br> FREE LIFESTYLE <br> 2022</h1>
+      <p class="intro-words">Peace of mind comes from knowing you and your family are covered and protected by the best
+        health insurance cover</p>
       <h1 id="intro3">50 YEARS IN BUSINESS </h1>
       <P id="intro4">We advise, Insure and Invest</P>
     </article>
-    <section class="row products-categories">
+    <section class="products-categories">
       <h3 class="products1">Main Products</h3>
 
-      <div class="product-life">
-        <img class="product-img" src="./imgs/life.jpeg" alt="Life Insurance">
-        <h2 class="product-name">Life</h2>
-        <p class="product-description">Our job is to help you choose the right insurance cover for you so that you may Live Free!</p>
-      </div>
+      <div class="row all-products">
+        <div class="product-life col-3" id="Life">
+          <img class="product-img" src="./imgs/life.jpeg" alt="Life Insurance">
+          <h2 class="product-name">Life</h2>
+          <p class="product-description">Our job is to help you choose the right insurance cover for you so that you may
+            Live Free!</p>
+        </div>
+
+        <div class="product-life col-3" id="Health">
+          <img class="product-img" src="./imgs/health.jpeg" alt="Health Insurance">
+          <h2 class="product-name">Health</h2>
+          <p class="product-description">Your family’s health and safety means the world to you. We understand that and
+            want to help
+            you protect your loved ones from situations beyond your control.</p>
+        </div>
 
-      <div class="product-life">
-        <img class="product-img" src="./imgs/health.jpeg" alt="Health Insurance">
-        <h2 class="product-name">Health</h2>
-        <p class="product-description">Your family’s health and safety means the world to you. We understand that and want to help 
-          you protect your loved ones from situations beyond your control.</p>
+        <div class="product-life col-3" id="General">
+          <img class="product-img" src="./imgs/matatu.jpeg" alt="Kenyan Matatu">
+          <h2 class="product-name">General</h2>
+          <p class="product-description">Let’s cover what you love and what you have worked hard to build.</p>
+        </div>
       </div>
+      <div class="get-quote">
+        <h2>Get Quote</h2>
+      </div>
+
+    </section>
+
+    <section class="management" id="management">
+      <h2 class="products1">Management</h2>
 
-      <div class="product-life">
-        <img class="product-img" src="./imgs/matatu.jpeg" alt="Kenyan Matatu">
-        <h2 class="product-name">General</h2>
-        <p class="product-description">Let’s cover what you love and what you have worked hard to build.</p>
+      <div class="row">
+        <div id="add-management"></div>
       </div>
```

Comment:
You're almost complete here but few additional changes to be made

- [ ] Kindly add the squared shapes at the top left of each management member photo to match the styles in the original design.
- [ ] Also, kindly work on the photos to not stretch. Currently, the images are stretching and look blurry. Please have all your images of the same with and height and not stretched as that will make your design classy.

**Your Design**
![image](https://user-images.githubusercontent.com/50941074/169516253-da27f1c0-61ef-4fa1-a47c-82f3fe0cdbaa.png)

**The Original Design**
![image](https://user-images.githubusercontent.com/50941074/169516387-13388e98-7057-4380-8e03-11c1284d8e5f.png)

Reply:
@AmaduKamara This has now been implimented